tests/acceptance/pageObjects/phoenixPage.js

```diff
@@ -52,6 +52,32 @@ module.exports = {
       return this.waitForElementPresent('@messageCloseIcon')
         .click('@messageCloseIcon')
         .waitForElementNotPresent('@messageCloseIcon')
+    },
+    getNotifications: async function () {
+      const notifications = []
+      await this
+        .click('@notificationBell')
+        .waitForElementVisible('@notificationElement')
+      await this.api.elements('@notificationElement', async result => {
+        result.value.map(item => {
+          notifications.push(new Promise((resolve) => {
+            this.api.elementIdText(item.ELEMENT, text => {
+              resolve(text.value)
+            })
+          })
+          )
+        })
+      })
+      return Promise.all(notifications)
+    },
+    assertNotificationIsPresent: async function (dataTable) {
+      let notificationIncludesDataTableElement
+      const dataTableHashed = dataTable.hashes()
+      const notifications = await this.getNotifications()
+      for (var i = 0; i < dataTableHashed.length; i++) {
+        notificationIncludesDataTableElement = notifications.includes(dataTableHashed[i].title)
```

Comment:
Can we rename `notificationIncludesDataTableElement` to something short and sweet? Maybe `isPresent`/`isVisible`?